File: src/main/java/com/ciazhar/authserver/model/dto/request/RegisterForm.java
package com.ciazhar.authserver.model.dto.request;

/**
 * Created by ciazhar on 6/23/17.
 */


public class RegisterForm {
    private String email;
    private String password;


    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    public String getPassword() {
        return password;
    }

    public void setPassword(String password) {
        this.password = password;
    }
}

File: src/main/kotlin/com/ciazhar/authserver/model/dto/request/RegisterForm.kt
package com.ciazhar.authserver.model.dto.request

import javax.validation.constraints.NotNull

/**
 * Created by ciazhar on 9/2/17.
 */

class RegisterForm(

        @field:NotNull
        var username : String?=null,

        @field:NotNull
        var email : String?=null,

        @field:NotNull
        var password : String?=null
)